### PR TITLE
Changed tagging resources route

### PR DESCRIPTION
**Pull request by SchneiderNicolas**
This PR changes the route to get resources.
Now using a POST Method instead of a GET Method.
The filter that allows to retrieve resources based on tags and missing tags is now implemented.

### CODE REVIEW

File: tagging/routes/routes.go [hunk 1]
```diff
@@ -74,13 +57,14 @@ func init() {
 
 func init() {
 	routes.MethodMuxer{
-		http.MethodGet: routes.H(routeGetResources).With(
+		http.MethodPost: routes.H(routeGetResources).With(
 			db.RequestTransaction{db.Db},
 			users.RequireAuthenticatedUser{users.ViewerAsParent},
-			routes.QueryArgs(resourcesQueryArgs),
+			routes.RequestContentType{"application/json"},
+			routes.RequestBody{ResourcesRequestBody{[]string{"394125495069"}, []string{"us-west-2"}, []string{"lambda"}, []Tag{{"project", "trackit"}}, []Tag{{"Product", "msol"}}}},
```
Comment (alexandre-snr) [inline tagging/routes/routes.go:74]: Is this supposed to be here ?

File: tagging/routes/routeGetResources.go [hunk 2]
```diff
@@ -101,38 +73,38 @@ func GetResources(ctx context.Context, params ResourcesQueryParams) (int, []util
...
-		} else if cast, ok := err.(*elastic.Error); ok && cast.Details != nil && cast.Details.Type == "search_phase_execution_exception" {
-			l.Error("Error while getting data from ES", map[string]interface{}{
-				"type":  fmt.Sprintf("%T", err),
-				"error": err,
-			})
-		} else {
-			l.Error("Query execution failed", map[string]interface{}{"error": err.Error()})
+func makeElasticSearchRequest(ctx context.Context, params ResourcesRequestBody, user users.User,
+	esSearchParams func(ResourcesRequestBody, *elastic.Client, string) *elastic.SearchService) (*elastic.SearchResult, int, error) {
+		l := jsonlog.LoggerFromContextOrDefault(ctx)
```
Comment (jklakosz) [inline tagging/routes/routeGetResources.go]: the body of this function is too indented, no?

File: tagging/routes/routes.go (hunk 1 above)
Comment (SchneiderNicolas) [reply tagging/routes/routes.go:74]: It's an example of request body.

Comment (alexandre-snr) [reply tagging/routes/routes.go:74]: :+1:

**Review by jklakosz — APPROVED** [review]

**Review by jklakosz — APPROVED** [review]

**Pull request merged**